release/rllib_tests/stress_tests/atari_impala_xlarge_tests.yaml

```diff
@@ -22,7 +22,7 @@ atari-impala:
         num_cpus_for_driver: 0
         rollout_fragment_length: 50
         train_batch_size: 500
-        num_workers: 128
+        num_workers: 127
```

Comment:
@kouroshHakha @gjoliver Makes it so that we can fit this test onto one 4GPU-32CPU instance and 5 96CPU instances which. Lmk if you have doubts.

Reply:
This is because of the limited set of GCE GPU instances currently offered by Anyscale.
There are other combinations of doing this.